src/stores/toolbarStore.ts

```diff
@@ -1,41 +1,76 @@
 import { observable, computed, action } from 'mobx';
+import _ from 'lodash';
 import ToolbarTool from '~/enums/toolbarTool';
 import EditMode from '~/enums/editMode';
 
+import BaseTool from '~/tools/BaseTool';
+import AddNetworkNodeTool from '~/tools/AddNetworkNodeTool';
+import AddNewRoutePathTool from '~/tools/AddNewRoutePathTool';
+import CopyTool from '~/tools/CopyTool';
+import DivideLinkTool from '~/tools/DivideLinkTool';
+import EditNetworkNodeTool from '~/tools/EditNetworkNodeTool';
+import PrintTool from '~/tools/PrintTool';
+
+const TOOLS = [
```

Comment:
Might be more simple to have object instead of array. That way there would be no need for `_.find`. Now it seems overly elaborate way of finding the tool specified.